sound/soc/sof/topology.c

```diff
@@ -317,6 +318,28 @@ static enum sof_ipc_dai_type find_dai(const char *name)
 	return SOF_DAI_INTEL_NONE;
 }
 
+struct sof_comp_domains {
```

Comment:
Does all of this need to belong in the common topology.c when this is IPC4 specific @jsarha

Reply:
Its just that all the other get_token_* functions are, but sure I can move these to ipc4-topology.c. Is that a good place for them?

Reply:
yes, I think so